### PR TITLE
add timeout and processes wait duration parameter to tests

### PR DESCRIPTION
**Pull request by francois-vincent**
this should contribute to get tests running on jenkins

### CODE REVIEW

**Comment by francois-vincent** [comment]
Jenkins, test this please

File: integration_tests/docker.py [hunk 1]
```diff
@@ -384,7 +385,11 @@ def start_services(self, *args, **kwargs):
                 else:
                     self.docker_exec('service {} start'.format(k), x)
         if wait:
-            self.wait_process(wait)
```
Comment (kinnou02) [inline integration_tests/docker.py]: wait is the pid of the process? I would choose a better name, at first for me it was related to --wait and wait_timeout

Comment (francois-vincent) [inline integration_tests/docker.py]: no it is a timeout

Comment (francois-vincent) [inline integration_tests/docker.py]: my bad, it is a process or a list of processes to wait for (like bash, rabbitmq, etc...)

Comment (kinnou02) [inline integration_tests/docker.py]: that was my point, it's not clear when reading the code :)

**Pull request merged**